packages/manager/modules/vps/src/dashboard/migrate/migrate.routing.js

```diff
@@ -9,8 +9,25 @@ export default /* @ngInject */ ($stateProvider) => {
       },
     },
     resolve: {
-      goToMigrateConfirm: /* @ngInject */ ($state) => () =>
-        $state.go('vps.detail.dashboard.migrate.confirm'),
+      migrationTrackingPrefix: /* @ngInject */ (vpsMigration) => {
```

Comment:
you are using this in multiple routes. Can you put it in parent route and reuse it?

Reply:
it is 2 different tracking prefixes which will be provided to single component so i am using it in 2 different places.